src/DynamicFormControl.vue

```diff
@@ -43,6 +47,20 @@
             formControl: Object,
             colWidth: String
         },
+        methods: {
+            isRed(value){
+                if (value){
+                    if(value.constructor === Array){
+                        if (value.length > 0){
```

Comment:
lines 54 to 57 can be simplified to `return value.length == 0` - but are empty arrays never valid values?

Reply:
If a value is required then presumably the user needs to select at least one value. If there is an option to explicitly set no values then that would still show as a value in the array anyway.

Reply:
yes you're absolutely right, that makes sense!